clang/lib/Tooling/Inclusions/Stdlib/StdSpecialSymbolMap.inc

```diff
@@ -411,6 +411,11 @@ SYMBOL(_27, std::placeholders::, <functional>)
 SYMBOL(_28, std::placeholders::, <functional>)
 SYMBOL(_29, std::placeholders::, <functional>)
 
+SYMBOL(binary_search, std::ranges::, <algorithm>)
```

Comment:
can you move this next to https://github.com/llvm/llvm-project/blob/2de1fc82861edbc484b7a1b82a37aa29d4b982de/clang/lib/Tooling/Inclusions/Stdlib/StdSpecialSymbolMap.inc#L363-L369 ?